### PR TITLE
Lyra 関連の処理を削除

### PR DESCRIPTION
**Pull request by miosakuma**
Lyra 関連の処理を削除しました。

確認したいこと
---

savedAssetsToLocal が不要と判断して全て削除したが理解はあっているか。

https://github.com/shiguredo/sora-unity-sdk-samples/compare/develop...feature/delete-lyra?expand=1#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL907-L941

---
This pull request primarily focuses on removing the Lyra-related processes from the codebase. The changes span across multiple files, including `CHANGES.md`, `.gitignore`, `SoraSample.cs`, and `copysdk.py`. The Lyra-related settings, file reading, and copying processes have been removed, along with the deletion of Lyra-related meta files and definitions.

Removal of Lyra-related processes:

* [`CHANGES.md`](diffhunk://#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R14-R20): Removed the Lyra-related processes and files from the change log.
* [`SoraUnitySdkSamples/.gitignore`](diffhunk://#diff-7526298b7eebb8bb1522d2931d9fc80a83d77dc40c4d79fa153022774612885eL89): Removed the Lyra-related file definition from the .gitignore file.
* [`SoraUnitySdkSamples/Assets/SoraSample.cs`](diffhunk://#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL1-R1): Removed the Lyra-related settings, file reading, and copying processes from the SoraSample.cs file. [[1]](diffhunk://#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL1-R1) [[2]](diffhunk://#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL78-L82) [[3]](diffhunk://#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL225-L227) [[4]](diffhunk://#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL591-L594) [[5]](diffhunk://#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL684-L695) [[6]](diffhunk://#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL728-L729) [[7]](diffhunk://#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL753-L756) [[8]](diffhunk://#diff-12695070cb2312ef563a9b63cc33a776880479ed95398221170c3c4e3876858bL907-L941)

Removal of Lyra-related files:

* [`SoraUnitySdkSamples/Assets/Plugins/SoraUnitySdk/ios/liblyra.a.meta`](diffhunk://#diff-31e2b3c2606d08df1471d44481369d46b8f1891a73ca5c31fe4f12e9f34fd409L1-L73): Removed the Lyra-related meta file.
* [`copysdk.py`](diffhunk://#diff-718a26fe90250da7d705fd6481ca51860be42080815c9f736bbaccca2e369fedL76-L83): Removed the Lyra-related file copying process from the copysdk.py file. [[1]](diffhunk://#diff-718a26fe90250da7d705fd6481ca51860be42080815c9f736bbaccca2e369fedL76-L83) [[2]](diffhunk://#diff-718a26fe90250da7d705fd6481ca51860be42080815c9f736bbaccca2e369fedL104-L106)

### CODE REVIEW

File: SoraUnitySdkSamples/Assets/SoraSample.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-using System.Collections;
+﻿using System.Collections;
```
Comment (melpon) [inline SoraUnitySdkSamples/Assets/SoraSample.cs:1]: これ何の diff だろう。BOM でも入ったのかな

Comment (miosakuma) [reply SoraUnitySdkSamples/Assets/SoraSample.cs:1]: すいません、git diff したら `^M` が入っていたようでした。
Mac で作業していて、Editor 上も LF で見えていて元に戻すのに苦労しております...

```
--- a/SoraUnitySdkSamples/Assets/SoraSample.cs
+++ b/SoraUnitySdkSamples/Assets/SoraSample.cs
@@ -1,4 +1,4 @@
-using System.Collections;
+using System.Collections;^M
```

**Review by melpon — APPROVED** [review]
よさそう

File: SoraUnitySdkSamples/Assets/SoraSample.cs (hunk 1 above)
Comment (miosakuma) [reply SoraUnitySdkSamples/Assets/SoraSample.cs]: BOM が混入していました a4c480b で削除しました。

**Pull request merged**